pkg/bootstrapper/bootstrapper.go

```diff
@@ -461,9 +485,9 @@ func (wmcb *winNodeBootstrapper) initializeKubeletFiles() error {
 	return nil
 }
 
-// getInitialKubeletArgs returns the kubelet args required during initial kubelet start up.
-// This function assumes that wmcb.kubeletArgs are populated.
-func (wmcb *winNodeBootstrapper) getInitialKubeletArgs() []string {
+// generateInitialKubeletArgs returns the kubelet args required during initial kubelet start up. args should be a map
```

Comment:
args -> Args

Reply:
I did lowercase since I was referring to the variable `args` which is lowercase